### PR TITLE
Update deploy

### PR DESCRIPTION
**Pull request by wesleymonte**

### CODE REVIEW

File: deploy/ansible-playbook/deploy-iguassu.yml [hunk 1]
```diff
@@ -1,13 +1,14 @@
...
-      apt: pkg=docker.io state=latest update_cache=yes cache_valid_time=3600
+      shell: "{{ install_script }}"
       become: yes
-- hosts: iguassu-machine
-  vars:
-    deploy_script_runner: bash deploy-docker.sh
-  tasks:
-    - name: Deploying iguassu in Iguassu Machine
-      shell: "{{ deploy_script_runner }}"
+    - name: Deploying Iguassu in Iguassu Machine
```
Comment (thiagomanel) [inline deploy/ansible-playbook/deploy-iguassu.yml]: "... to the Iguassu Machine"

File: deploy/scripts/deploy-docker.sh [hunk 2]
```diff
@@ -3,26 +3,28 @@
 DIR_PATH=$(pwd)
 CONF_FILES_DIR="conf-files"
 
-IMAGE_NAME="ufcglsd/iguassu"
+IMAGE_NAME="ufcglsd/iguassu:latest"
 CONTAINER_NAME="iguassu"
 
-MANAGER_PORT="8080"
-CONTAINER_PORT="9000"
+MANAGER_FRONT_PORT="9000"
```
Comment (thiagomanel) [inline deploy/scripts/deploy-docker.sh]: are these ports usually defined in bash or in a conf file?

Comment (wesleymonte) [reply deploy/scripts/deploy-docker.sh]: In the bash. Ports are defined by the application, so they are already used in bash.

Comment (thiagomanel) [reply deploy/scripts/deploy-docker.sh]: sounds odd. Anyway, I'll approve.

Comment (emanueljoivo) [reply deploy/scripts/deploy-docker.sh]: It's a better practice remove the hardcoded configuration from scripts: more flexibility and customization without changing the source code. Move definition ports and the private key file name to a conf file.

File: deploy/scripts/deploy-docker.sh [hunk 3]
```diff
@@ -3,26 +3,28 @@
 DIR_PATH=$(pwd)
 CONF_FILES_DIR="conf-files"
 
-IMAGE_NAME="ufcglsd/iguassu"
+IMAGE_NAME="ufcglsd/iguassu:latest"
 CONTAINER_NAME="iguassu"
```
Comment (emanueljoivo) [inline deploy/scripts/deploy-docker.sh]: In the near future, we'll use various docker containers/images, only one per application component, (e.g. one container to the dashboard, other to the backend). I find it useful to leave these fields as configurable. Move the container/images definitions to a conf file.

File: deploy/install.sh [hunk 4]
```diff
@@ -1,51 +1,99 @@
 #!/bin/bash
 
-DIR_PATH=$(pwd)
+MY_PATH="`dirname \"$0\"`"              # relative
```
Comment (emanueljoivo) [inline deploy/install.sh]: Improve scripting comments for more meaningful phrases.

**Pull request merged**